pkg/server/server.go

```diff
@@ -0,0 +1,157 @@
+// Copyright 2017 Envoyproxy Authors
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+// Package server provides an implementation of a streaming xDS server
+package server
```

Comment:
nit: package server isn't a great name for a package, especially when the type that it holds is also called server; server.server is repetitive. 

My suggestion is to combine this file and the previous one into a package called grpc, so you have a grpc.cache and a grpc.server.

Reply:
Moved server to `grpc` package. I think cache is intended to be where all the state is managed between handlers independent of the transport issues. Not sure if it should be part of grpc yet.